hw/ip/pinmux/pinmux_reg.core

```diff
@@ -8,10 +8,11 @@ filesets:
   files_rtl:
     depend:
       - lowrisc:tlul:headers
-
+      - "fileset_top ? (lowrisc:systems:pinmux_reg)"
```

Comment:
@msfschaffner do you mind double checking this part?

Reply:
this looks good to me